django_lti_tool_provider/tests/test_views.py

```diff
@@ -133,36 +141,20 @@ def test_no_session_given_incorrect_payload_throws_bad_request(self, _):
         self.assertIn("Invalid LTI Request", response.content)
 
     def test_no_session_correct_payload_processes_lti_request(self, patched_send_lti_received):
-        with self.assertRaises(LtiUserData.DoesNotExist):
-            LtiUserData.objects.get(user=self.user)  # precondition check
+        # Precondition check
+        self.assertFalse(LtiUserData.objects.all())
 
         response = self.send_lti_request(self.get_correct_lti_payload())
 
-        self._verify_lti_created(self.user, self._data)
-        self._verify_redirected_to(response, self._url_base + self.DEFAULT_REDIRECT)
-        self._verify_lti_updated_signal_is_sent(patched_send_lti_received, self.user)
-
-    @ddt.data('GET', 'POST')
-    def test_session_set_processes_lti_request(self, method, patched_send_lti_received):
```

Comment:
@e-kolpakov At this point they should be forcibly authenticated, so it shouldn't apply anymore, yes?